addons/web/static/src/scss/fields.scss

```diff
@@ -446,6 +458,53 @@
 
 }
 
+.o_field_invalid {
+    .o_signature {
+        outline: 3px solid #DC6965;
+        cursor: pointer;
+    }
+}
+.o_form_editable {
+    .o_signature:hover {
+        outline: 3px solid #00A09D;
+        cursor: pointer;
+    }
+}
+
+.card.o_sign_font_dialog {
+    @include o-position-absolute(0, 0);
+    display: none;
+    height: 100%;
+
+    .o_sign_font_selection {
+        overflow: auto;
+        height: 90%;
+        padding: 0;
+
+        > a {
+            height: 100px;
+            padding: 0;
+
+            > img {
+                height: 100%;
+            }
+
+            > span {
+                font-size: 4rem;
+                overflow:hidden;
+                white-space:nowrap;
+            }
+        }
+    }
+}
+
+.o_sign_signature_draw .o_sign_signature {
+    padding: 0;
+    > canvas.jSignature {
+        outline: #ced4da 1px solid;
```

Comment:
try to not hardcode colors in the styles, make them depend on an existing variable